testgrid/config.yaml

```diff
@@ -3823,67 +3817,79 @@ dashboards:
       alert_mail_to_addresses: kubernetes-sig-cluster-lifecycle@googlegroups.com
     alert_stale_results_hours: 24
     num_failures_to_alert: 2 # Runs every 12h. Alert when it's been failing for 24 hours
-  - name: kubeadm-kind-master
-    test_group_name: ci-kubernetes-e2e-kubeadm-kind-master
+  - name: kubeadm-kind-1.15
+    test_group_name: ci-kubernetes-e2e-kubeadm-kind-1-15
+    alert_options:
+      alert_mail_to_addresses: kubernetes-sig-cluster-lifecycle@googlegroups.com
```

Comment:
#13205 is a change to migrate everyone to a testgrid annotation, but all of the annotations here are currently supported, [right now](https://github.com/kubernetes/test-infra/blob/master/testgrid/config.md)

If this has a corresponding Prow Job to move it to, then it won't be moved out of this file in the near future

Reply:
i will have the annotations in this PR.
thanks to the annotations and with some smart (but not so obvious) naming of test_groups and planning we can even avoid touching testgrid/config.yaml for approval, although the current limitations around security job approval for presubmits leaves us with pinging the same folks (top-level approvers).